doc/integrator/fulltext_search.rst

```diff
@@ -289,3 +289,33 @@ And define the configuration in the ``vars.yaml`` file:
 
 ``fr: fr`` is a link between the pyramid language and the text search configuration, by default it is
 ``fr: french`` because the default french text search configuration is named 'french'.
+
+
+Don't use Synonym and Thesaurus Dictionary
+------------------------------------------
+
+With those dictionaries e.-g. 'alignement' is simplifyed as 'align':
```

Comment:
We recommend to avoid using the Synonym and Thesaurus dictionaries, because when these are used, a word like 'alignement' is simplified as 'align':

Reply:
It's not a recommendation, it's a choice...

Reply:
Ah, yes. So we should say that. Something like;

You may wish to avoid using the Synonym and Thesaurus dictionaries, because when these are used, a word like 'alignement' is simplified as 'align':